### DISCUSSION TITLE
Self-Hosted Runner -- Log File

### DISCUSSION THREAD

**Discussion by dhandalianai** [community/community#25538] [Actions]
<p>I’m new to GitHub Actions.  I’ve set up a self-hosted runner as a service on my EC2 instance (per <a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-the-self-hosted-runner-application-as-a-service" rel="nofollow noopener">here</a>) and am wondering if there’s a log file that the service is writing on my machine?  If so, where can I find it?  Thanks!</p>

**Comment by BrightRan** [comment, marked as answer]
<p>Hi <a class="mention" href="/dhandalianai">@dhandalianai</a> ,</p>
<p>You can get the log files from the folder <strong>_diag</strong> under the runner installation directory. From the log files you can view the diagnostic logs for recently run workflows.</p>
<p><img src="https://user-images.githubusercontent.com/46307996/181091753-17bd91a5-2d75-4271-8bdc-80b0d23d9e96.png" alt="folder.PNG" data-base62-sha1="aSnvXCxeC2QIi8EK0KZF6Ky8S4w" width="145" height="162"></p>

**Comment by dhandalianai** [comment]
<p>Thanks <a class="mention" href="/brightran">@brightran</a> !  Exactly what I needed, appreciate your reply. </p>
